src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -440,16 +444,28 @@ def unregistered_message
       end
 
       # Informative message
+      # @param registered [Boolean] is the system registered?
       # @return [String] translated message
-      def media_upgrade
-        # TRANSLATORS: Media based upgrade requested by user (1/2)
+      def media_upgrade(registered)
+        # TRANSLATORS: Media based upgrade requested by user (1/3)
         #   User requested media based upgrade which does not use SCC/SMT
         #   but the downloaded media (physical DVD or shared repo on a local server).
-        _("<h2>Media Based Upgrade</h2><p>The media based upgrade is requested. " \
+        ret = _("<h2>Media Based Upgrade</h2><p>The media based upgrade is requested. " \
           "In this mode YaST will not contact the registration server to obtain " \
           "the new software repositories required for migration.</p>") +
-          # TRANSLATORS: Media based upgrade requested by user (2/2)
+          # TRANSLATORS: Media based upgrade requested by user (2/3)
           _("<p>Please add the installation media manually in the next step.</p>")
+
+        return ret unless registered
+
+        # TRANSLATORS: a warning message, upgrading the registered systems
+        #   using media is not supported
+        ret + _("<h2>Warning!</h2><p><b>The media based upgrade for registered " \
+          "systems is not supported!<b></p>") +
+          _("<p>If you upgrade the system using media the registration status " \
+            "will not be upgraded and the system will be still registered " \
```

Comment:
np: I would say that "will not be updated" (the status). But I guess Ken will review the texts anyway...